LICENSE

```diff
@@ -1,201 +1,116 @@
-      incurred by, or claims asserted against, such Contributor by reason
-      of your accepting any such warranty or additional liability.
-
-   END OF TERMS AND CONDITIONS
-
-   APPENDIX: How to apply the Apache License to your work.
-
-      To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "[]"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright [yyyy] [name of copyright owner]
-
-   Licensed under the Apache License, Version 2.0 (the "License");
-   you may not use this file except in compliance with the License.
-   You may obtain a copy of the License at
-
-       http://www.apache.org/licenses/LICENSE-2.0
-
-   Unless required by applicable law or agreed to in writing, software
-   distributed under the License is distributed on an "AS IS" BASIS,
-   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-   See the License for the specific language governing permissions and
-   limitations under the License.
+                        Aspect Community License
+
+BY DOWNLOADING, COPYING, OR OTHERWISE USING THE SOFTWARE WITH WHICH THIS LICENSE AGREEMENT IS
+PROVIDED (THE “SOFTWARE”), YOU OR THE ENTITY YOU REPRESENT (“LICENSEE”) ARE CONSENTING TO BE BOUND
+BY AND ARE BECOMING A PARTY TO THIS LICENSE AGREEMENT (“AGREEMENT”).
+
+IF YOU DO NOT AGREE TO ALL OF THE TERMS OF THIS AGREEMENT, THEN YOU MAY NOT DOWNLOAD THE SOFTWARE
+AND MUST DELETE ANY COPIES THAT YOU HAVE ALREADY DOWNLOADED.
+
+IF LICENSEE IS AN ENTITY, YOU REPRESENT AND WARRANT THAT YOU ARE AUTHORIZED TO BIND LICENSEE.
+IF THESE TERMS ARE CONSIDERED AN OFFER, ACCEPTANCE IS EXPRESSLY LIMITED TO THESE TERMS.
+
+1.	Grant.
+
+    Subject to the terms of this Agreement, Aspect Build Systems, Inc. (“Aspect”)
+    hereby grants Licensee (and only Licensee) a limited, non-sublicensable, non-transferable,
+    royalty-free, nonexclusive license to use the Software only in Licensee’s organization and only
+    in accordance with any documentation that accompanies it.
+    
+    The Software may only be used by a Licensee who is:
+    (i)   an individual (and only for personal use),
+    (ii)  a Small Business (as defined below),
+    (iii) a non-profit entity or an academic or university institution, or
+    (iv)  a current or former Aspect Customer (as defined below).
+    
+    A “Small Business” is any entity with fewer than 50 total employees (including, for purposes
+    of such calculation, all employees of any entities affiliated with such entity).
+
+    An “Aspect Customer” is any entity that has remitted payment to Aspect at any time.
```

Comment:
Explicitly include entities that have contributed to our Open Collective?

Reply:
That would be nice, but contributions appear on there as "Guest" and "Finn" on https://opencollective.com/aspect-build/projects/protobuf-and-grpc, "Troy Carlson" and "Tim Jaeger" on https://opencollective.com/aspect-build, etc.

When companies have donated, almost none of the engineers went through the extra steps to setup a corporate presence on OpenCollective.

So we'd have to list these in another way, and keep that list updated and make it legally bound to the license grant.